CSV file 'dataset.csv' already exists.
(27)_20

(27)_20
srun: Job 7671275 step creation temporarily disabled, retrying (Requested nodes are busy)
srun: Step created for job 7671275

(27)_20

(27)_20

(27)_20

(27)_20
srun: Job 7671275 step creation temporarily disabled, retrying (Requested nodes are busy)
srun: Step created for job 7671275

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

Plot saved to Plots/archer2/7671275/refall_cpuvalidate_27_0_performance.png
Plot saved to Plots/archer2/7671275/func1_cpuvalidate_27_0_performance.png
(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

Plot saved to Plots/archer2/7671275/refall_cpuvalidate_27_1_performance.png
Plot saved to Plots/archer2/7671275/func1_cpuvalidate_27_1_performance.png
(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20
srun: Job 7671275 step creation temporarily disabled, retrying (Requested nodes are busy)
srun: Step created for job 7671275

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

Plot saved to Plots/archer2/7671275/refall_cpuvalidate_27_2_performance.png
Plot saved to Plots/archer2/7671275/func1_cpuvalidate_27_2_performance.png
(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

Plot saved to Plots/archer2/7671275/refall_cpuvalidate_27_3_performance.png
Plot saved to Plots/archer2/7671275/func1_cpuvalidate_27_3_performance.png
(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

(27)_20

Plot saved to Plots/archer2/7671275/refall_cpuvalidate_27_4_performance.png
Plot saved to Plots/archer2/7671275/func1_cpuvalidate_27_4_performance.png
Function 1 and 2 execution time measured
Traceback (most recent call last):
  File "/mnt/lustre/a2fs-work1/work/d35/d35/weiyu24/Microbenchmarking-of-Accelerators-Offloading/openmpbench_C_v40/Analysis/analysis.py", line 137, in <module>
    main()
  File "/mnt/lustre/a2fs-work1/work/d35/d35/weiyu24/Microbenchmarking-of-Accelerators-Offloading/openmpbench_C_v40/Analysis/analysis.py", line 126, in main
    with open(args.data_filename, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'Analysis/archer2/7671275/data_Function 1 and 2 execution time measured.txt'
Function 1 execution time measured
Traceback (most recent call last):
  File "/mnt/lustre/a2fs-work1/work/d35/d35/weiyu24/Microbenchmarking-of-Accelerators-Offloading/openmpbench_C_v40/Analysis/analysis.py", line 137, in <module>
    main()
  File "/mnt/lustre/a2fs-work1/work/d35/d35/weiyu24/Microbenchmarking-of-Accelerators-Offloading/openmpbench_C_v40/Analysis/analysis.py", line 126, in main
    with open(args.data_filename, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'Analysis/archer2/7671275/data_Function 1 execution time measured.txt'
Function 1 execution time subtracted
Traceback (most recent call last):
  File "/mnt/lustre/a2fs-work1/work/d35/d35/weiyu24/Microbenchmarking-of-Accelerators-Offloading/openmpbench_C_v40/Analysis/analysis.py", line 137, in <module>
    main()
  File "/mnt/lustre/a2fs-work1/work/d35/d35/weiyu24/Microbenchmarking-of-Accelerators-Offloading/openmpbench_C_v40/Analysis/analysis.py", line 126, in main
    with open(args.data_filename, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'Analysis/archer2/7671275/data_Function 1 execution time subtracted.txt'
Function 2 execution time measured
Traceback (most recent call last):
  File "/mnt/lustre/a2fs-work1/work/d35/d35/weiyu24/Microbenchmarking-of-Accelerators-Offloading/openmpbench_C_v40/Analysis/analysis.py", line 137, in <module>
    main()
  File "/mnt/lustre/a2fs-work1/work/d35/d35/weiyu24/Microbenchmarking-of-Accelerators-Offloading/openmpbench_C_v40/Analysis/analysis.py", line 126, in main
    with open(args.data_filename, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'Analysis/archer2/7671275/data_Function 2 execution time measured.txt'
